pallets/nfts/src/features/approvals.rs

```diff
@@ -165,10 +179,197 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		Self::deposit_event(Event::AllApprovalsCancelled {
 			collection,
-			item,
+			item: Some(item),
 			owner: details.owner,
 		});
 
 		Ok(())
 	}
+
+	/// Approves the transfer of items in the collection that owned by the origin to a delegate.
```

Comment:
/// Approves the collection items owned by the origin to a delegate.

Reply:
This way of phrasing might help to simplify the explanation of collection approvals. This needs to be improved throughout the pallet.